### PR TITLE
Add a github munger.

### PR DESCRIPTION
**Pull request by brendandburns**
Copied over from https://github.com/kubernetes/kubernetes/pull/12365

### CODE REVIEW

File: mungegithub/pulls/blunderbuss.go [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+		}
+	}
+	return owners.List()
+}
+
+type BlunderbussMunger struct {
+	config *BlunderbussConfig
+}
+
+var blunderbussConfig = flag.String("blunderbuss-config", "", "Path to the blunderbuss config file")
```
Comment (mikedanese) [inline mungegithub/pulls/blunderbuss.go]: as a future addition, the blunderbuss-config could be loaded from the repository lake the .travis.yml

**Comment by mikedanese** [comment]
Looks cool. Mungers should declare (maybe in a comment) which actions should trigger a remunge e.g. on creation, on comments, on new pushes... Then we trigger munges from github hooks API.

File: mungegithub/pulls/blunderbuss.go (hunk 1 above)
Comment (brendandburns) [inline mungegithub/pulls/blunderbuss.go]: seems reasonable, though much trickier to do than including it in the image ;)

**Comment by brendandburns** [comment]
@mikedanese thanks for the ideas for future improvements.  I can haz lgtmz?

**Comment by mikedanese** [comment]
LGTM.

**Pull request merged**